pom.xml

```diff
@@ -41,7 +41,8 @@
                 <projectName>${package-name}</projectName>
                 <projectDescription>${package-description}</projectDescription>
                 <pythonicPackageName>ansys-grantami-recordlists</pythonicPackageName>
-                <pythonicPackageUrl>https://github.com/pyansys/grantami-recordlists</pythonicPackageUrl>
+                <pythonicPackageUrl>https://github.com/ansys/grantami-recordlists</pythonicPackageUrl>
+                <homepageUrl>https://github.com/ansys/grantami-serverapi-openapi</homepageUrl>
```

Comment:
In retrospect, we probably only want the homepage to be populated on the PyPI page (since all links go to the same place). So I've only specified the homepage link.